src/services/openklant2/service.ts

```diff
@@ -458,9 +507,26 @@ export async function postActor({
 }
 
 export const saveKlantContact = async (
-  data: KlantContactPostmodel,
+  vraag: Vraag,
 ): Promise<SaveKlantContactResponseModel> => {
-  const response = await postKlantContact(data);
+  const klantcontactPostModel: KlantContactPostmodel = {
+    kanaal: vraag.kanaal,
+    onderwerp:
+      vraag.vraag?.title === "anders"
```

Comment:
ik zie dat dit stuk verplaatst is van contactmomentafhandeling.vue, maar misschien kunnen we deze logica iets duidelijker opschrijven dan met genested ternaries?  Ik vind het op deze manier lastig te volgen wat de logica precies is omdat ik niet snel zie hoe 'diep' ik zit. Ik vind zoiets duidelijker, maar dat is subjectief. Laat maar weten wat jij vindt.
```ts
function getOnderwerp(vraag: Vraag): string {
  if(vraag.vraag.title === "anders") {
    return vraag.specifiekevraag
  }
  
  if(vraag.vraag.title && vraag.specifiekevraag) {
    return `${vraag.vraag.title} (${vraag.specifiekevraag})`
  }

  return vraag.vraag.title || vraag.specifiekevraag
}
```

Reply:
hier kunnen we een richtlijn over afspreken. voor nu laat ik het staan akkoord?